lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -108,13 +108,15 @@ abstract class BaseAdapter {
 
   final DomRenderer domRenderer;
   PointerDataCallback _callback;
-  Map<int, bool> _isDownMap = <int, bool>{};
-  bool _isButtonDown(int button) {
-    return _isDownMap[button] == true;
+  Map<int, Map<int, bool>> _isDownMap = <int, Map<int,bool>>{};
```

Comment:
Let's add a comment that explains what we put in this map of maps

Reply:
Done.